exercises/Strings-3-R.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: exercise
+title: Strings 3
+language: R
+---
+
+The length of an organism is typically strongly correlated with its body
+mass. This is useful because it allows us to estimate the mass of an
+organism even if we only know its length. This relationship generally
+takes the form `Mass (kg) = a * Length(m)^b`, where the parameters `a` and
+`b` vary among groups. Write a script that prompts the user for the
+following pieces of information:
+
+1. Genus name
+2. Species name
+3. The length of the species
+
+and then estimates the mass of the organism using the equation above.
+The script should print out the result using formatted printing as:
+
+*Genus* *species* is *length* meters long and weighs approximately *mass* kg.
+
+where the words in ***italics*** are replaced with the appropriate
+values. As is standard practice the first letter (and only the first
```

Comment:
As I mentioned in the solution, let's just drop the capitalization requirement since it's so tricky in R.